ui/ducks/send/send.js

```diff
@@ -1006,13 +1014,19 @@ const slice = createSlice({
           recipient.error = null;
         }
 
-        if (
-          isSendingToken &&
-          isValidHexAddress(recipient.userInput) &&
-          (toChecksumAddress(recipient.userInput) in contractMap ||
-            checkExistingAddresses(recipient.userInput, tokens))
-        ) {
-          recipient.warning = KNOWN_RECIPIENT_ADDRESS_WARNING;
+        if (isSendingToken && isValidHexAddress(recipient.userInput)) {
+          // token from dynamic api list is fetched when useStaticTokenList is true
+          const userInput = useStaticTokenList
+            ? recipient.userInput
+            : toChecksumAddress(recipient.userInput);
```

Comment:
I don't follow why we Checksum the input if useStaticTokenList is false?

Reply:
When the flag is false the tokenList contains tokens from `contract-metadata`, and the address of tokens in `contract-metadata` are checksum addresses.

Reply:
But then you shouldn't need to cast them to checkSum again? So If anything it should be the other way around right? And it doesn't hurt to checkSum an address that is already formatted as such. I think we should remove the ternary? @ryanml any thoughts on this?

Reply:
ping here @ryanml @NiranjanaBinoy @Gudahtt

Reply:
Ah sorry @NiranjanaBinoy I understand this now. Could you add a comment explaining why the ternary is necessary?